packages/nasm.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Nasm < Package
+  version '2.14rc16'
+  description 'The Netwide Assembler'
+  homepage 'https://www.nasm.us/'
```

Comment:
I know it doesn't really matter from Chromebrew's perspective but can you please put the version below the homepage to keep it consistent for humans? :)

Reply:
OK.